### PR TITLE
Use tempio to generate APK packages

### PR DESCRIPTION
**Pull request by ludeeus**
Centralize APK list and use [tempio](https://github.com/home-assistant/tempio) to generate the 2 formats we need.

### CODE REVIEW

**Comment by dgomes** [comment]
Should this PR update .github/workflows/ci.yml too ?

File: test [hunk 1]
```diff
@@ -0,0 +1 @@
+autoconf automake bluez-dev build-base cmake cups-dev eudev-dev ffmpeg-dev git glib-dev gmp-dev libffi-dev libjpeg-turbo-dev libpng-dev libxml2-dev libxslt-dev linux-headers mpc1-dev mpfr-dev openssl-dev postgresql-dev tiff-dev
```
Comment (dgomes) [inline test:1]: Probably this needs to be deleted

Comment (ludeeus) [reply test:1]: uuups :D

**Review by dgomes — APPROVED** [review]

File: requirements.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+	"apk": [
```
Comment (frenck) [inline requirements.json:2]:
```suggestion
    "apk": [
```

File: requirements.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        "libxslt-dev",
+        "linux-headers",
+        "linux-headers",
+        "mpc1-dev",
+        "mpfr-dev",
+        "musl-dev",
+        "openssl-dev",
+        "postgresql-dev",
+        "tiff-dev"
+	]
```
Comment (frenck) [inline requirements.json:28]:
```suggestion
    ]
```

**Pull request merged**